index.js

```diff
@@ -1,8 +1,142 @@
 const express = require('express');
+const crypto = require('crypto');
+var exec = require('child_process').execFile;
```

Comment:
This could be confusing because there is a function in the child_process module called "exec" (distinct from "execFile"). I would change variable to have a different name.